tests/unit/completion/test_models.py

```diff
@@ -269,29 +273,31 @@ def test_url_completion(qtmodeltester, config_stub, web_history, quickmarks,
 
     Verify that:
         - quickmarks, bookmarks, and urls are included
-        - no more than 'web-history-max-items' history entries are included
+        - no more than 'web-history-max-items' items are included (TODO)
```

Comment:
If we have lazy loading, we might be able to ditch `web-history-max-items` altogether without performance issues.